src/components/table/Table.tsx

```
@@ -339,15 +339,19 @@ export function Table({
     },
     ...columns,
-    {
-      id: DROPDOWN_COL_ID,
-      cell: (props: any) => (
-        <TableActions actionsConfig={actionsConfig} data={props.row.original} />
-      ),
-      header: 'Actions',
-      size: 70,
-      enablePinning: true,
-    },
+    ...(!!Object.values(rowSelection).length
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)